### PR TITLE
[15 Min Fix] Added the custom Cypress command, createResponseTemplate, to create comment templates.

### PR DESCRIPTION
**Pull request by nickytonline**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update
- [x] Testing infrastructure

## Description

Adds a Cypress custom command for creating canned responses.

## Related Tickets & Documents

#13149

## QA Instructions, Screenshots, Recordings

N/A  This is testing infrastructure

### UI accessibility concerns?

N/A  This is testing infrastructure.

## Added tests?

- [ ] Yes
- [x] No, and this is why: This is testing infrastructure
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [x] I've updated the [Developer Docs](https://docs.forem.com) and/or
      [Admin Guide](https://forem.gitbook.io/forem-admin-guide/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] I've updated the README or added inline documentation
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [ ] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] What gif best describes this PR or how it makes you feel?

Captain America catching his shield

### CODE REVIEW

File: docs/tests/e2e-tests.md [hunk 1]
```diff
@@ -144,6 +145,18 @@ cy.createArticle({
 });
 ```
 
+### Creating a Canned Response Command
+
+To create a canned as part of your test's setup, use the `cy.cannedResponse`
```
Comment (aitchiss) [inline docs/tests/e2e-tests.md:150]:
```suggestion
To create a canned response as part of your test's setup, use the `cy.cannedResponse`
```

File: cypress/support/commands.js [hunk 2]
```diff
@@ -196,3 +196,22 @@ Cypress.Commands.add(
     });
   },
 );
+
+/**
+ * Creates a canned response.
+ *
+ * @param {string} title The title of a canned response.
+ * @param {string} [content=''] The content of the canned response.
```
Comment (aitchiss) [inline cypress/support/commands.js:204]:
```suggestion
 * @param {string} content The content of the canned response.
```

**Review by aitchiss — APPROVED** [review]
just a couple of small suggestions but looks good to me 👍

**Review by rhymes — COMMENTED** [review]
I wonder if we can avoid adding another "domain concept". As we don't use the word `canned` anywhere in the code, maybe we can keep the expression "response template" even in Cypress

**Comment by nickytonline** [comment]
> I wonder if we can avoid adding another "domain concept". As we don't use the word `canned` anywhere in the code, maybe we can keep the expression "response template" even in Cypress

@rhymes, I'll make the change.

**Review by Zhao-Andy — APPROVED** [review]
LGTM!

**Review by juliannatetreault — APPROVED** [review]

**Pull request merged**